core/src/test/scala/org/apache/spark/status/protobuf/KVStoreProtobufSerializerSuite.scala

```diff
@@ -31,6 +34,32 @@ import org.apache.spark.ui.scope.{RDDOperationEdge, RDDOperationNode}
 class KVStoreProtobufSerializerSuite extends SparkFunSuite {
   private val serializer = new KVStoreProtobufSerializer()
 
+  test("All the string fields must be optional to avoid NPE") {
```

Comment:
This test checks the protobuf file to make sure all the string fields are defined as `optional string`

Reply:
Example failure output:
```
ArrayBuffer(("  string storage_level = 5;", 267), ("  string name = 1;", 363)) was not empty
All the string fields should be defined as `optional string` for handling null string.
Please update the following fields:
line #267:   string storage_level = 5;
line #363:   string name = 1;
```